rust/src/dcerpc/dcerpc.rs

```diff
@@ -170,6 +170,7 @@ pub struct DCERPCTransaction {
     pub ctxid: u16,
     pub opnum: u16,
     pub first_request_seen: u8,
+    pub version: String,
```

Comment:
can we do this simpler? IIRC version is just 5.0 or 5.1 or similar? So it could be expressed as (u8,u8) or even as a single u8 (50, 51), etc.